src/test/isolation2/expected/lockmodes.out

```diff
@@ -2182,111 +2182,6 @@ UPDATE 1
 ROLLBACK
 1q: ... <quitting>
 
-<<<<<<< HEAD
-=======
```

Comment:
Why did the conflict resolution choose the current version instead of incoming version ? Did we lose this test in the `.sql`?

Reply:
Yes, my bad, in previous PR #858 I forget to clean them.
The cases is related to GDD which generates wrong results. Seems won't be fixed in CBDB in a short time https://github.com/apache/cloudberry/issues/54

Reply:
CBDB doesn't have that cases in .sql, may be removed intentionally. The case in .out is re-introduced by cherrry-pick because GPDB modify that case.

Reply:
okay.